html-relative-links/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href= "index.html">Home Page</a></li>
+    <li><a href= "about-me/about-me.html">About Me</a></li>
+    <li><a href= "projects/projects.html">Projects</a></li>
+    <li><a href= "contact-us/contact-us.html">Contact Us</a></li>
```

Comment:
No spaces around equal signs for HTML attributes, please

Reply:
done and pushed!

<img width="1440" alt="Screen Shot 2021-10-05 at 11 48 53 AM" src="https://user-images.githubusercontent.com/88061673/136084568-201f4cff-b6e5-4c0b-a6b5-d555d9f347ef.png">